core/src/test/java/edu/wpi/grip/core/sources/HttpSourceTest.java

```diff
@@ -57,7 +57,7 @@ public void setUp() throws URIException, URISyntaxException {
         contextStore,
         GripServer.IMAGE_UPLOAD_PATH);
 
-    logoFile = new File(Files.class.getResource("/edu/wpi/grip/images/GRIP_Logo.png").toURI());
+    logoFile = new File(Files.class.getResource("/edu/wpi/grip/images/old_GRIP_Logo.png").toURI());
```

Comment:
This should probably just reference the `util.Files` static variable instead of having the same string twice in the tests. Can you change this?